database/seed.js

```diff
@@ -11,14 +11,14 @@ const shirts = [];
 
 for( let i = 1; i <= 100; i++) {
   shirts.push ({
-    ProductId: i,
-    ImageURLs: []
+    productId: i,
+    imageURLs: []
   });
 }
 
 for (let url of imageUrls) {
```

Comment:
With how this is run, is it possible the some shirts will not have a imageURL?

Reply:
Yes, each shirt will have a random number of images, and that might mean some have no images.